gsa/src/web/pages/audits/dialog.js

```diff
@@ -56,6 +61,44 @@ import Layout from 'web/components/layout/layout';
 import AddResultsToAssetsGroup from 'web/pages/tasks/addresultstoassetsgroup';
 import AutoDeleteReportsGroup from 'web/pages/tasks/autodeletereportsgroup';
 
+const get_scanner = (scanners, scanner_id) => {
```

Comment:
IMHO we are using camelCase now
```suggestion
const getScanner = (scanners, scanner_id) => {
```